assets/scss/modules/_datatables.scss

```diff
@@ -12,31 +13,15 @@
   }
 }
 
-#js-select-all {
-  width: 3%;
-}
-
-.controlpanel {
-  background-color: $white;
-  width: 100%;
-  padding: 5px 0;
-
-  &.fixed {
-    position: fixed;
-    top: 0;
-    z-index: 999;
-  }
-}
-
-.controlpanel__items {
-  position: relative;
+.dataTables_filter__block {
```

Comment:
the single underscore in this name doesn't follow BEM syntax. should either be a hyphen or a double underscore.